data/test/news/c0000000002o.json

```diff
@@ -70,7 +78,7 @@
                   "height": 2952,
                   "locator": "/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png",
                   "originCode": null,
-                  "copyrightHolder": "BBC"
+                  "copyrightHolder": "Getty Images"
```

Comment:
I'd suggest making this change on another test article, for example `c0000000005o`, since the images on this article have another logo in the bottom right.
Also, could you please update this table in the Readme, explaining that there are images with Image source attribution on the associated articles: https://github.com/bbc/simorgh/blob/latest/data/README.md

Reply:
Changed to `c0000000005o` & updated readme.